### ISSUE TITLE
Missing Varieties for Burmy.

### ISSUE THREAD

**Issue by YumYummity** [PokeAPI/api-data#173]
Burmy's default form is Plant Cloak, which is there, however it appears to be missing the Trash Cloak and Sandy Cloak.

[https://raw.githubusercontent.com/PokeAPI/api-data/master/data/api/v2/pokemon-species/412/index.json](https://raw.githubusercontent.com/PokeAPI/api-data/master/data/api/v2/pokemon-species/412/index.json)

The evolution, Wormadam, has all the cloaks.

**Comment by Deleca7755** [comment]
I don't think It Is supposed to have varieties since It's completely cosmetic, like Shellos, https://raw.githubusercontent.com/PokeAPI/api-data/master/data/api/v2/pokemon-species/433/index.json who also doesn't have more than one variety listed

It's forms are listed In the `forms` key of the `pokemon` resource https://raw.githubusercontent.com/PokeAPI/api-data/master/data/api/v2/pokemon/412/index.json
